### PR TITLE
[@types/three] Fix small errors in geometries of three

### PR DESCRIPTION
**Pull request by Methuselah96**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://stripe.com/docs/api/node#refund_object-failure_reason
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a tslint.json containing { "extends": "dtslint/dt.json" }.

### CODE REVIEW

**Review by XanderLuciano — CHANGES_REQUESTED** [review]
Couple parameters appear to be in the wrong order.

File: types/three/three-core.d.ts [hunk 1]
```diff
@@ -7492,7 +7511,7 @@ export class TorusGeometry extends Geometry {
 }
 
 export class TorusKnotBufferGeometry extends BufferGeometry {
-    constructor(radius?: number, tube?: number, radialSegments?: number, tubularSegments?: number, p?: number, q?: number, heightScale?: number);
+    constructor(radius?: number, tube?: number, radialSegments?: number, tubularSegments?: number, p?: number, q?: number);
```
[the remark this answers is not in the document]
Comment (Methuselah96) [reply types/three/three-core.d.ts]: Good catch. Fixed.

File: types/three/three-core.d.ts [hunk 2]
```diff
@@ -7506,7 +7525,7 @@ export class TorusKnotBufferGeometry extends BufferGeometry {
 }
 
 export class TorusKnotGeometry extends Geometry {
-    constructor(radius?: number, tube?: number, radialSegments?: number, tubularSegments?: number, p?: number, q?: number, heightScale?: number);
+    constructor(radius?: number, tube?: number, radialSegments?: number, tubularSegments?: number, p?: number, q?: number);
```
[the remark this answers is not in the document]
Comment (Methuselah96) [reply types/three/three-core.d.ts]: Fixed.

**Review by XanderLuciano — APPROVED** [review]
LGTM 👍

**Pull request merged**